permissions/plugin-preflight-integration.yml

```diff
@@ -0,0 +1,8 @@
+---
+name: "preflight-integration"
+github: "jenkinsci/preflight-integration-plugin"
+paths:
+- "org/jenkins-ci/plugins/preflight-integration"
```

Comment:
```suggestion
- "io/jenkins/plugins/preflight-integration"
```

Reply:
This should be the groupId from your pom.xml: https://github.com/jenkinsci/preflight-integration-plugin/blob/master/pom.xml#L10